src/app/components/event-page/event-page.component.html

```diff
@@ -0,0 +1,84 @@
+<div id="event-card"> 
+    <div class="flex-row" id="top-div">
+        <img id="event-image" [src]="transformStringToImage() ? transformStringToImage() : ''" alt="">
+        <div id="event-information" class="flex-column">
+            <div class="flex-row" id="event-information-first-row">
+                <div  id="event-text-information" class="flex-column">
+                    <div id="title-date">
+                        <div id="event-title" class="bold-element">{{event.title}}</div>
+                        <div id="start-date">{{transformDate(event.startingDate)}}</div>
```

Comment:
https://angular.io/api/common/DatePipe
Use a datePipe from Angular to display the date in a different format.
It's not a good practice to use functions inside the template; those will get evaluated on each time Angular triggers change detection.
https://javascript.plainenglish.io/angular-dont-call-a-function-inside-the-template-anymore-e74ebf499bb8